card/mod/05_standard/set/type/signup.rb

```diff
@@ -96,7 +96,8 @@ def has_token?
 end
 
 event :activate_account do
-  subcards['+*account'] = {'+*status'=>'active'}
+  add_subfield :account
+  subfield(:account).add_subfield :status, :content =>'active'
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator '=>'.